pkgs/servers/uwsgi/default.nix

```diff
@@ -92,6 +93,14 @@ stdenv.mkDerivation rec {
     inherit python2 python3;
   };
 
+  # Documentation is maintained separately, without any releases.
+  sphinxRoot = fetchFromGitHub {
```

Comment:
How much does it increase the closure size? Perhaps this behavior can be changed via a `.override` argument ?